src/v2/Apps/Onboarding/OnboardingApp.tsx

```diff
@@ -0,0 +1,108 @@
+import { FC } from "react"
+import { Text, Button, Flex, Spacer, Box } from "@artsy/palette"
+import {
+  OnboardingProvider,
+  useOnboardingContext,
+} from "./useOnboardingContext"
+import { useRouter } from "v2/System/Router/useRouter"
+import { OnboardingWelcome } from "./Views/OnboardingWelcome"
+import { OnboardingQuestionOne } from "./Views/OnboardingQuestionOne"
+import { OnboardingQuestionTwo } from "./Views/OnboardingQuestionTwo"
+import { OnboardingDone } from "./Views/OnboardingDone"
+import {
+  VIEW_AUCTION_HIGHLIGHTS,
+  VIEW_CURATED_ARTWORKS,
+  VIEW_SEARCH_ARTISTS,
+  VIEW_SEARCH_ARTWORKS,
+  VIEW_SEARCH_GALLERIES,
+  VIEW_TRENDING_ARTISTS,
+  VIEW_TRENDING_LOTS,
+} from "./config"
+import { OnboardingProgress } from "./Components/OnboardingProgress"
+
+export const OnboardingApp: FC = () => {
+  const { router } = useRouter()
+
+  const handleDone = () => {
+    router.push("/")
+  }
+
+  return (
+    <OnboardingProvider onDone={handleDone}>
+      <OnboardingDebug />
+
+      <Spacer mt={2} />
+
+      <OnboardingSteps />
+    </OnboardingProvider>
+  )
+}
+
+const OnboardingSteps: FC = () => {
+  const { current, next } = useOnboardingContext()
```

Comment:
non-blocking suggestion: `current` / `next` would be friendlier if they were like `currentStep`, `nextStep`